autopush/db.py

```diff
@@ -582,6 +582,9 @@ def register_user(self, data):
         conn = self.table.connection
         db_key = self.encode({"uaid": hasher(data["uaid"])})
         del data["uaid"]
+        if "router_type" not in data or "connected_at" not in data:
+            # Not specifying these values will generate an exception in AWS.
+            return (False, {}, data)
```

Comment:
Is this going to accidentally mask a call that thinks the condition is failing vs. being called with incorrect data?